script/elevate-cpanel.PL

```diff
@@ -928,20 +930,18 @@ responsible of the multiple reboots.
 
 sub run_stage_1 ($self) {
 
-    # do not leave cruft behing when aborting
-    my $abort = sub {
-        $self->do_cleanup();
-        exit 42;    ## no critic(Cpanel::NoExitsFromSubroutines) catching signals
-    };
+    Elevate::Marker::startup();
 
-    local $SIG{'INT'} = $abort;
-    local $SIG{'HUP'} = $abort;
+    Elevate::Motd->setup();
 
-    return 1 unless _sanity_check();
+    if ( !$self->should_run_leapp() ) {
+        Elevate::Stages::bump_stage();
+        return ACTION_CONTINUE;
+    }
 
-    print_box( "Starting stage 1 of " . VALID_STAGES . ": Installing " . $self->service->name . " service" );
+    $self->run_component_once( 'Grub2' => 'mark_cmdline' );
```

Comment:
FIX:  We should skip this if they provide the no-leapp option since it has the potential to die

Reply:
Already handled in the block above this by `should_run_leapp`.

Reply:
ah, missed that.